hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/contracts/operation/HederaCreateOperationExternalizerTest.java

```diff
@@ -274,6 +263,9 @@ void hasExpectedHollowAccountCompletionOnSuccessWithoutSidecarEnabled() {
         given(creator.createSuccessfulSyntheticRecord(any(), any(), any())).willReturn(liveRecord);
         given(dynamicProperties.enabledSidecars()).willReturn(Set.of());
         given(childFrame.getContractAddress()).willReturn(PRETEND_CONTRACT_ADDRESS);
+        final var newContractMock = mock(Account.class);
+        given(newContractMock.getNonce()).willReturn(1L);
```

Comment:
Good to have a test of the nonce but `1L` is not a good choice since that's what it _used_ to be hardwired to!  Pick a different number! (`2L` would be fine!)